dex/models/trades/blast/platforms/blasterswap_blast_base_trades.sql

```diff
@@ -0,0 +1,66 @@
+{{
+    config(
+        schema = 'blasterswap_blast',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{% set evt_sources = [
```

Comment:
Is this something specific to balsterswap, do they only support these coins or are we missing decoding for Pair generally_

Reply:
We may missing decoding for Pair generally.
But blasterswap v2 is only limited to these 4 pools:
https://github.com/blasterswap/blasterswap-core-v2?tab=readme-ov-file#pairs

Reply:
nice find